efficientdet/keras/postprocess.py

```diff
@@ -214,10 +214,10 @@ def postprocess_combined(params, cls_outputs, box_outputs, image_scales=None):
           score_threshold=score_thresh,
           clip_boxes=False))
   nms_classes += CLASS_OFFSET
+  nms_boxes = clip_boxes(nms_boxes, params['image_size'])
```

Comment:
Why clip should happen before scaling the box? I thought we need to first scale and then clip?